ann_benchmark/src/vespa/ann_benchmark/vespa_ann_benchmark.cpp

```diff
@@ -180,7 +203,8 @@ HnswIndex::find_top_k(uint32_t k, const std::vector<float>& value, uint32_t expl
      * read guard is not taken here.
      */
     TopKResult result;
-    TypedCells typed_cells(&value[0], CellType::FLOAT, value.size());
+    std::vector<float> normalized_value;
```

Comment:
I think we should keep this vector as a member variable to avoid creating it per search.

Reply:
That would cause incorrect behavior for batch mode.

Reply:
Good point.